### ISSUE TITLE
Improve plugin options

### ISSUE THREAD

**Issue by ThmX** [taigaio/taiga-contrib-slack#44]
Hi, We use the slack integration at work and everything is working awesome. However one tiny problem is that it is either too quiet or too noisy, by that I mean

I would propose a better way to tune the notification by splitting the `change` option in multiple options.
At the very least, I would see `minor change` (description, option) and `major change` (comments, status, assign).

PS: if it is interesting for you, I can try to dive in the plugin code in order to implement it.

**Comment by bameda** [comment]
Of course, all contributions are welcome but If you need to change the interface please send us you proposal before.

**Comment by alejandrobarrios07** [comment]
+1

**Comment by arisjr** [comment]
+1

**Comment by arisjr** [comment]
Hi, I could say more. A noisy plugin will be abandoned by its users. 

"Change" is the most important option, but "Change" is to chatty. In our case, I think the most important changes could be text input from taiga users, like comments and notes/block notes. Statuses changes could be added too. But nothing other than that. 

On our case, taiga slack plugin was dominating the slack chat room, so it was realocated to a secundary channel, that no one access. Damn. It's so important but it's being taken aside. 
I want slack plugin to be on the most important channel, but for that it can only generate important messages for us.

This is what is happening on my job. Perhaps this is occurring on other teams.

Thanks and regards.
